addons/mail/static/src/models/thread_view.js

```diff
@@ -89,7 +89,7 @@ registerModel({
          */
         _computeCallView() {
             return (this.thread && this.thread.model === 'mail.channel' && this.thread.rtcSessions.length > 0)
-                ? insertAndReplace()
+                ? insertAndReplace({ thread: replace(this.thread) })
```

Comment:
Remove this.

If you need `thread` in `CallView`, just use `callView.threadView.thread`.
Or if you want to just make `callView.thread`, just define a related field `CallView/thread` which relates to `CallView/threadView . ThreadView/thread`